x-pack/plugin/esql/qa/testFixtures/src/main/resources/topN.csv-spec

```diff
@@ -0,0 +1,68 @@
+
+ host:keyword  | card:keyword  |                                ip0:ip                                           |                         ip1:ip
+gamma          |eth0           |fe80::cae2:65ff:fece:feb9                                                        |127.0.0.3                                             
+gamma          |lo0            |fe80::cae2:65ff:fece:feb9                                                        |fe81::cae2:65ff:fece:feb9                             
+epsilon        |eth0           |[fe80::cae2:65ff:fece:feb9, fe80::cae2:65ff:fece:fec0, fe80::cae2:65ff:fece:fec1]|fe80::cae2:65ff:fece:fec1                             
+epsilon        |eth1           |null                                                                             |[127.0.0.1, 127.0.0.2, 127.0.0.3]                     
+epsilon        |eth2           |[fe81::cae2:65ff:fece:feb9, fe82::cae2:65ff:fece:fec0]                           |[fe81::cae2:65ff:fece:feb9, fe82::cae2:65ff:fece:fec0]
+beta           |eth0           |127.0.0.1                                                                        |::1                                                   
+beta           |eth1           |127.0.0.1                                                                        |128.0.0.1                                             
+beta           |eth1           |127.0.0.1                                                                        |127.0.0.2
+;
+
+ipSortingByTwoIPs
+from hosts | sort ip0, ip1 desc | limit 8 | keep host, card, ip0, ip1;
+
+ host:keyword  | card:keyword  |                                ip0:ip                                           |                         ip1:ip
+alpha          |eth1           |::1                                                                              |::1                      
+beta           |eth1           |127.0.0.1                                                                        |128.0.0.1                
+beta           |eth1           |127.0.0.1                                                                        |127.0.0.2                
+alpha          |eth0           |127.0.0.1                                                                        |127.0.0.1                
+beta           |eth0           |127.0.0.1                                                                        |::1                      
+gamma          |lo0            |fe80::cae2:65ff:fece:feb9                                                        |fe81::cae2:65ff:fece:feb9
+epsilon        |eth0           |[fe80::cae2:65ff:fece:feb9, fe80::cae2:65ff:fece:fec0, fe80::cae2:65ff:fece:fec1]|fe80::cae2:65ff:fece:fec1
+gamma          |eth0           |fe80::cae2:65ff:fece:feb9                                                        |127.0.0.3
+;
+
+ipSortingBySameIPfield
+from hosts | sort ip0 asc nulls first, ip0 desc, card, host, ip1 | limit 15 | keep host, card, ip0, ip1 | where card != "lo0";
```

Comment:
Note that is not only "sorting once by ip0", is first sorting `asc` then `desc` by `ip0`. With the current query you get

```
...
epsilon        |eth0           |[fe80::cae2:65ff:fece:feb9, fe80::cae2:65ff:fece:fec0, fe80::cae2:65ff:fece:fec1]|fe80::cae2:65ff:fece:fec1                             
gamma          |eth0           |fe80::cae2:65ff:fece:feb9                                                        |127.0.0.3
...
```

and I believe that the order of these two rows specifically will be different if the sorting changes (either `asc` `desc` is changed or only one sorting is used) because the true sorting value of `[fe80::cae2:65ff:fece:feb9, fe80::cae2:65ff:fece:fec0, fe80::cae2:65ff:fece:fec1]` will change based on its `min` / `max` values. Then if you take the `min` / `max` of this row and compare to the second row value - `fe80::cae2:65ff:fece:feb9` - it will sort differently for `min` than `max.